### DISCUSSION TITLE
Known incompatibility with 22621

### DISCUSSION THREAD

**Discussion by apthorpe** [valinet/ExplorerPatcher#1090] [General]
From https://support.microsoft.com/en-us/topic/march-14-2023-kb5023706-os-build-22621-1413-9d3f2de5-08e7-4462-8fba-d944201f4ae1 in the Known Issues in this Update list, second from the bottom:

"After installing this or later updates, Windows devices with some third-party UI customization apps might not start up. These third-party apps might cause errors with explorer.exe that might repeat multiple times in a loop. The known affected third-party UI customization apps are ExplorerPatcher and StartAllBack. These types of apps often use unsupported methods to achieve their customization and as a result can have unintended results on your Windows device."

I am seeing the behavior described.

Edition	Windows 11 Pro
Version	22H2
Installed on	‎2022-‎12-‎02
OS build	22621.1413
Serial number	PF45NZMP
Experience	Windows Feature Experience Pack 1000.22639.1000.0

I am still working to resolve this issue; regardless, the patch notes from MS explicitly call out ExplorerPatcher; you may want to look into this further.

**Comment by rys322** [comment]
I am having this issue and am stuck in an infinite loop of explorer crashing and restarting. Any advice would be desperately appreciated.

**Reply by topia** [reply]
https://github.com/valinet/ExplorerPatcher/issues/1089

**Reply by peseoane** [reply]
In my case I have finished the process explorer.exe to fix it.

Then I launched "control.exe" and uninstalled Explorer Patcher.

At the moment everything is working again, but what a chore...

**Comment by Kirimoar** [comment]
Hmm, there is another discussion on another thread, but this here is ofc also interesting.

I faced the same problem yesterday - as much as I love the ExplorerPatcher, this was REALLY a huge problem. Took me more than an hour to finally fix it (after identifying what even the problem was).

Solution was a rollback to the windows-update before.

However - I was not running the latest EP version when the bug happened.
Does anybody know if this still happens if I'm going to update windows again?
And what could be a solution?
